core/src/main/se/tevej/game/libgdx/view/rendering/input/KeyBoardLibgdxAdapter.java

```diff
@@ -14,6 +14,10 @@
 
     static final Map<Integer, TButton> inputMap = new HashMap<>();
 
+    /**
```

Comment:
We should not explain a static block. The comment should talk about the mapping between TKey and libGDXs keycodes